packages/babel-core/test/errors-stacks.js

```diff
@@ -0,0 +1,292 @@
+import * as babel from "../lib/index.js";
+
+// TODO: Remove this in Babel 8, once we drop Node.js 8
```

Comment:
```suggestion
```
Seems like this should be removed at the same time, I didn't understand it when I submitted it just now.